src/main/resources/db/changelog/data/financialTransaction.csv

```diff
@@ -0,0 +1,9 @@
+id,wallet_id,transaction_type,amount,transaction_date,description
+1,1,INCOME,15000,2022-02-19T09:47:52.595721658,""
```

Comment:
`"`  `"` is treated as a String and not as EMPTY_STRING.
It is possible by SQL but it's not possible by Liquibase.
So this line is not needed.
However, this is not a problem. The front-end, when it finds a NULL value - that is, the user did not provide a description value - will insert the EMPTY_STRING value as `"` `"`.